tests/wpt/metadata/xhr/abort-during-upload.any.js.ini

```diff
@@ -1,9 +1,8 @@
 [abort-during-upload.any.html]
   [XMLHttpRequest: abort() while sending data]
-    expected: FAIL
+    expected: PASS
```

Comment:
All of the test results that now pass can be removed entirely. If an ini file has no more `expected:` lines, then the entire file can be removed.